File: 75-SortColors/75-SortColors.py
# Last updated: 12/17/2025, 10:54:07 AM
class Solution(object):
    def sortColors(self, nums):
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
        for i in range(len(nums)):
            j=i+1
            while j<len(nums):
                if nums[j]<nums[i]:
                    nums[i], nums[j] = nums[j], nums[i]
                j+=1
        return nums

File: 75-SortColors/75-SortColors.java
// Last updated: 12/17/2025, 10:54:02 AM
class Solution {
    public void sortColors(int[] nums) {
        for(int i=0; i<nums.length; i++){
            int j = i+1;
            while (j<nums.length){
                if (nums[j]<nums[i]){
                    int temp = nums[j];
                    nums[j] = nums[i];
                    nums[i] = temp;
                }
                j++;
            }
        }
    }
}